### ISSUE TITLE
Load Remote Module 

### ISSUE THREAD

**Issue by amer201326** [manfredsteyer/module-federation-plugin-example#24]
Hi, 
can I use the (loadRemoteModule) function to load a component from React project?
i try this :

` const r = await loadRemoteModule({`
`            type: 'script',`
`            remoteName:'react',`
`            remoteEntry: 'http://localhost:4204/remoteEntry.js',`
`            exposedModule: './web-components'`
 `       });`

`        const ref2 = this.reactContainer.createComponent(r.react);`

but it not work,

or any way to dynamic load React component and pass parameter to it ?

**Comment by mannb1** [comment]
Yes, I would love to see this example expanded to have a React Remote as well.  I am trying to adapt your other example but the lerna dependency has me confused - I much prefer this example without the lerna dependency.  Any chance of adding a React remote to this example?
